src/include/rho/FixedVector.hpp

```diff
@@ -215,6 +215,7 @@ namespace rho {
 	    // this object, so account for it here.
 	    size_t bytes = (size() - 1) * sizeof(T);
 	    MemoryBank::adjustBytesAllocated(-bytes);
+            GCNode::adjustFreedSize(sizeof(FixedVector), bytes);
```

Comment:
The most common value of size() is 1.  In that case we should skip the calls to adjust size.